src/CC-BY-4.0.xml

```diff
@@ -35,7 +35,7 @@
          the licensed material may still be restricted for other reasons, including because others have
          copyright or other rights in the material. A licensor may make special requests, such as asking that
          all changes be marked or described. Although not required by our licenses, you are encouraged to
-         respect those requests where reasonable. More considerations for the public.</p>
+         respect those requests where reasonable. More<alt match=" |_"> </alt>considerations for the public<alt match=": wiki.creativecommons.org/Considerations_for_licensees|\.">: wiki.creativecommons.org/Considerations_for_licensees</alt></p>
```

Comment:
This and other `<alt>` elements in this PR will need a `name` attribute to comply with 0d0e50c (#519) (see also #518, which has downstream tooling complaining about a missing `name`).